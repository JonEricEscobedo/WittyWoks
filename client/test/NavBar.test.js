import React from 'react';
import NavBar from '../../client/src/components/NavBar.jsx';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';


it('renders correctly', () => {
  const tree = renderer.create(
    <NavBar />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

describe('A suite to test the Resume component', () => {

  it('Expect NavBar.jsx to render without throwing an error', () => {
    expect(shallow(<NavBar />).contains(<a className="navbar-brand" href="/"><strong>BestFit</strong></a>)).toBe(true);
  });
});
