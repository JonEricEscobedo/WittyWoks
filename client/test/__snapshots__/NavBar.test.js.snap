// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Navigation"
>
  <nav
    className="navbar navbar-toggleable-sm navbar-dark fixed-top transition"
  >
    <div
      className="container"
    >
      <button
        aria-controls="navbarNav1"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler navbar-toggler-right"
        data-target="#navbarNav1"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <a
        className="navbar-brand"
        href="/"
      >
        <strong>
          BestFit
        </strong>
      </a>
      <div
        className="collapse navbar-collapse"
        id="navbarNav1"
      >
        <ul
          className="navbar-nav ml-auto"
        >
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#home"
            >
              Home 
              <span
                className="sr-only"
              >
                (current)
              </span>
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#about"
            >
              About Us
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#offer"
            >
              What We Offer
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link"
              href="#contact"
            >
              Contact
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="btn btn-primary btn-sm"
              href="/auth/google"
              id="menu-button"
            >
              <i
                aria-hidden="true"
                className="fa fa-google"
              />
               Log In
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
</div>
`;
